site/charts/index.html

```
@@ -0,0 +1,173 @@
+          <div
+            style="display:grid;grid-template-columns:repeat(auto-fit,minmax(280px,1fr));gap:16px"
+          >
+            <article class="card">
+              <span class="tag">Tremor</span>
+              <h3 class="t-h3" style="margin:14px 0 8px">Prop-first components</h3>
+              <p>
+                Line, area, bar, donut, and bar list demos expose readable categories, nullable
+                gaps, value formatters, and metadata for drilldown.
+              </p>
+              <pre class="code-block" style="margin-top:16px;font-size:12px;overflow:auto">const wide = toEngineWideTableModel(result, {
+  seriesLabel: (s) => s.labels.get("host") ?? "host",
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)